### ISSUE TITLE
Ubuntu-Ubuntu with PMS 1.12.1.4885 not working

### ISSUE THREAD

**Issue by IvanCharcos** [wnielson/Plex-Remote-Transcoder#73]
Does not seem to be working with this setup.

I followed everything in the Ubuntu-Ubuntu tutorial but the streams are still being transcoded by the master.

Please let me know if there anything I can provide to ease debugging.

**Comment by liviynz** [comment]
Hi,

The readme me (kind of unclearly) states that the current PRT hasn’t been fully tested to be supported with the current version of Plex...however I run it and it works ok for me. The most common problem for people with installing the newer versions of Plex is they turn on hardware encoding and PRT won’t function with that turned on due to the way that they’ve coded their software. It’s not meant to work the way we made it run across multiple systems and we can trick the software into running when it’s using CPU for the workload but graphics hardware is a very different and ridged beast unless the developers included some standard components which would have allowed it...which they didn’t unfortunately. 

TLDR - in the Plex server settings, go to transcoder and untuck “use hardware acceleration when available” and try again.

**Comment by IvanCharcos** [comment]
Unfortunately, that one didnt solve the problem.

**Comment by IvanCharcos** [comment]
I ran prt check_config and got this:

`Host 192.168.0.50
plex@192.168.0.50's password: 
  Connect: OK
  Path: '/mnt/bigvol0/Media/TV Shows'
plex@192.168.0.50's password: 
Traceback (most recent call last):
  File "/usr/local/bin/prt", line 9, in <module>
    load_entry_point('prt==0.4.4', 'console_scripts', 'prt')()
  File "build/bdist.linux-x86_64/egg/prt.py", line 722, in main
  File "build/bdist.linux-x86_64/egg/prt.py", line 580, in check_config
ValueError: need more than 0 values to unpack
`

**Comment by IvanCharcos** [comment]
Cleaned up a bit and made sure all directories are set correctly (mountpoints/permissions) as well as network connectivity working.

Still nothing.

Now my local Plex transcoding wont even work properly..

**Comment by nydave69** [comment]
Have you figured out your issue..  I just noticed this, i Had 1.12.0 working,

**Comment by IvanCharcos** [comment]
I never figured it out. I ended up just installing PMS in a docker on the main transcoding server.

**Comment by nydave69** [comment]
Ok was just hoping you weren't still struggling with it..

**Comment by IvanCharcos** [comment]
Thank you. My friends/family were bugging me about it so I had no choice but to start the server somewhere else! =)

**Comment by liviynz** [comment]
To be closed. User implemented alternate solution.
